### PR TITLE
Add RFC2119 definition

### PR DESCRIPTION
**Pull request by williamkapke**
Defining "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"

This is pretty common thing to have.

### CODE REVIEW

**Comment by refack** [comment]
+1

**Comment by bnb** [comment]
If this is a change to the `GOVERNANCE.md` file, do we need to get Board approval @williamkapke?

**Comment by williamkapke** [comment]
@bnb yes. It looks like I only mention that on #182. Sry.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -92,5 +92,7 @@ If an agenda item cannot reach a consensus a CC member can call for either a clo
 - Adviser: a Collaborator within a Community Project elected to represent the Community Project on the CC.
 - Member: a person who participates in the development of the Community Committee through code or other artifacts.
 
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in all Community Committee documents are to be interpreted as described in RFC 2119.
```
Comment (dshaw) [inline GOVERNANCE.md]: Link to https://www.ietf.org/rfc/rfc2119.txt

**Comment by bnb** [comment]
@williamkapke what do we need to do to surface this change to the board? 🤔

**Comment by williamkapke** [comment]
We need consensus and I'll add it to the Board agenda for approval. It's pretty easy.

File: GOVERNANCE.md (hunk 1 above)
Comment (williamkapke) [reply GOVERNANCE.md]: @dshaw Link added.

**Comment by williamkapke** [comment]
@bnb I don't see any objections... so I'll send a request to add it to the Dec 18th meeting.

I'll land this PR after the Board approves it.

**Comment by bnb** [comment]
Sounds good - Thanks @williamkapke

**Comment by MylesBorins** [comment]
@williamkapke was there a resolution on this at the board?

**Comment by williamkapke** [comment]
Yes. It was approved!

**Pull request merged**